### PR TITLE
[test]Modify the testcase with query

### PR DESCRIPTION
**Pull request by jingkl**
Signed-off-by: jingkl <jingjing.jia@zilliz.com>
issue:#16833

### CODE REVIEW

File: tests/python_client/testcases/test_query.py [hunk 1]
```diff
@@ -259,7 +259,11 @@ def test_query_expr_non_primary_fields(self):
         res = df.iloc[:2, :-1].to_dict('records')
         for field in non_primary_field:
             filter_values = df[field].tolist()[:2]
-            term_expr = f'{field} in {filter_values}'
+            if field is not (ct.default_string_field_name):
```
Comment (binbinlv) [inline tests/python_client/testcases/test_query.py:262]: This bracket may be deleted.

File: tests/python_client/testcases/test_query.py [hunk 2]
```diff
@@ -1279,12 +1282,26 @@ def test_query_string_expr_with_binary(self):
...
                            check_task=CheckTasks.check_query_results, check_items={exp_res: res})
+    
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_query_string_with_invaild_prefix_expr(self):
+        """
+        target: test query with 
+        method: specify string primary field, use invaild prefix string expr
+        expected: raise error
+        """
+        collection_w, vectors = self.init_collection_general(prefix, insert_data=True)[0:2]
```
Comment (binbinlv) [inline tests/python_client/testcases/test_query.py:1302]: I think this should be better: "collection_w = self.init_collection_general(prefix, insert_data=True)[0]"

**Comment by jingkl** [comment]
Related to pr：#17129

File: tests/python_client/testcases/test_query.py (hunk 1 above)
Comment (jingkl) [reply tests/python_client/testcases/test_query.py]: Thanks, already modified

File: tests/python_client/testcases/test_query.py (hunk 2 above)
Comment (jingkl) [reply tests/python_client/testcases/test_query.py]: Thank you, already modified

**Pull request closed without merging**